ResearchKit/Common/ORKReviewStepViewController.m

```diff
@@ -0,0 +1,243 @@
+/*
+ Copyright (c) 2015, Oliver Schaefer.
+ 
+ Redistribution and use in source and binary forms, with or without modification,
+ are permitted provided that the following conditions are met:
+ 
+ 1.  Redistributions of source code must retain the above copyright notice, this
+ list of conditions and the following disclaimer.
+ 
+ 2.  Redistributions in binary form must reproduce the above copyright notice,
+ this list of conditions and the following disclaimer in the documentation and/or
+ other materials provided with the distribution.
+ 
+ 3.  Neither the name of the copyright holder(s) nor the names of any contributors
+ may be used to endorse or promote products derived from this software without
+ specific prior written permission. No license is granted to the trademarks of
+ the copyright holders even if such marks are included in this software.
+ 
+ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+ FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+
+#import "ORKReviewStepViewController.h"
+#import "ORKReviewStep.h"
+#import "ORKStep_Private.h"
+#import "ORKTaskViewController_Internal.h"
+#import "ORKStepViewController_Internal.h"
+#import "ORKSkin.h"
+#import "ORKTableContainerView.h"
+#import "ORKStepHeaderView_Internal.h"
+#import "ORKNavigationContainerView_Internal.h"
+#import "ORKTextChoiceCellGroup.h"
+
+
+typedef NS_ENUM(NSInteger, ORKReviewSection) {
+    ORKReviewSectionSpace1 = 0,
+    ORKReviewSectionAnswer = 1,
+    ORKReviewSectionSpace2 = 2,
+    ORKReviewSectionCount
+};
+
+@interface ORKReviewStepViewController () <UITableViewDataSource, UITableViewDelegate>
+
+@property (nonatomic, strong) ORKTableContainerView *tableContainer;
+@property (nonatomic, strong) UITableView *tableView;
+@property (nonatomic, strong) ORKStepHeaderView *headerView;
```

Comment:
Not sure if it is an good to keep those references: `tableView` and `headerView`, which can be accessed from `tableContainer`; Easy to make them out of synced.